commit 457d86d79a1496014b45a5bc6b14c23c4bdf4fa9
Author: Kurt Hindenburg <kurt.hindenburg@gmail.com>
Date:   Sat Aug 8 09:49:10 2015 -0400

    Export HistoryScroll to fix compiling issue with gcc 5.2
    
    REVIEW: 123881
    (cherry picked from commit 5f58169ba6fb18887b0e98aa390496df6fe8f242)

diff --git a/src/History.h b/src/History.h
index 6314ef9..dcd3819 100644
--- a/src/History.h
+++ b/src/History.h
@@ -83,7 +83,7 @@ private:
 //////////////////////////////////////////////////////////////////////
 class HistoryType;
 
-class HistoryScroll
+class KONSOLEPRIVATE_EXPORT HistoryScroll
 {
 public:
     explicit HistoryScroll(HistoryType*);
